src/stores/api/api.store.tsx

```diff
@@ -1,15 +1,29 @@
+import axios from "axios";
 import getClass from "data/helpers/classHelper";
 import { alignmentList, DnDClass } from "data/model/dndModel";
-import { action, Action } from "easy-peasy";
+import { action, Action, thunk, Thunk } from "easy-peasy";
 
 export interface APIModel {
   classStat: Record<string, any> | null;
+  classes: Record<string, any> | null;
+  class: Record<string, any> | null;
```

Comment:
What's the difference between `classes` and `class`?

Reply:
classes is a list of all classes (12 of them). index, name, url
class is all the data associated with a single class. name, hit_points, skills, etc.

Reply:
Got you 💯